{"name": "test_failed_fixture_value_in_display_name", "status": "passed", "description": "\n    >>> import allure\n    >>> import pytest\n\n    >>> @pytest.fixture\n    ... def fix():\n    ...     raise AssertionError(\"Fixture failed for some reason\")\n\n    >>> @allure.title('title with {fix}')\n    ... def test_fixture_value_name(fix):\n    ...     pass\n    ", "attachments": [{"name": "stdout", "source": "7892a9e0-75c1-4573-b402-801ee8d10ffa-attachment.txt", "type": "text/plain"}], "start": 1710872965595, "stop": 1710872965618, "uuid": "ef46faca-213a-4aca-9aba-ded962fec50e", "historyId": "5fafb55adb957b8c89d626b8c11a0211", "testCaseId": "5fafb55adb957b8c89d626b8c11a0211", "fullName": "tests.allure_pytest.acceptance.display_name.display_name_test#test_failed_fixture_value_in_display_name", "labels": [{"name": "parentSuite", "value": "tests.allure_pytest.acceptance.display_name"}, {"name": "suite", "value": "display_name_test"}, {"name": "host", "value": "MacBook-Pro-whenessel.local"}, {"name": "thread", "value": "29518-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.allure_pytest.acceptance.display_name.display_name_test"}]}